app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @comment = Comment.new(comment_params)
+    @comment.user = @user
+    @comment.post = @post
```

Comment:
- [ ] Same thing here as I pointed in the previous comment, Kindly make use of the `current_user` method created in the `ApplicationController` to set the user for a comment

  Example:
```rb
    @post = @user.posts.find(params[:post_id])
    @comment = current_user.comments.new(comment_params)
    @comment.post = @post
```

_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_

Reply:
Thank you, @LionRouge1 I'll look into all those items